apps/experiments/tasks.py

```diff
@@ -49,6 +49,7 @@ def get_response_for_webchat_task(
 @shared_task
 def get_prompt_builder_response_task(team_id: int, user_id, data_dict: dict) -> dict[str, str | int]:
     llm_service = LlmProvider.objects.get(id=data_dict["provider"]).get_llm_service()
+    llm_provider_model = LlmProviderModel.objects.get(id=data_dict["model"])
```

Comment:
Can do it in a future PR, but we should update the frontend to send `model_id` instead of `model` if this is what it really is.

Reply:
https://github.com/dimagi/open-chat-studio/pull/849